src/govuk/components/accordion/_index.scss

```diff
@@ -87,6 +89,73 @@
       }
     }
 
+    // Create Chervon icon aligned with text
+    .govuk-accordion-nav__chevron {
+      box-sizing: border-box;
+      display: inline-block;
+
+      position: relative;
+
+      width: 20px;
+      height: 20px;
+
+      border: 1px solid;
+      border-radius: 50%;
+
+      vertical-align: text-top;
+
+      // IE8 fallback of icon
+      @include govuk-if-ie8 {
+        display: inline-block;
+        max-height: 20px;
+        line-height: 1;
+      }
+
+      // Create inner Chervon arrow
+      &:after {
+        content: "";
+        box-sizing: border-box;
+        display: block;
+
+        position: absolute;
+        bottom: 5px;
+        left: 6px;
+
+        width: 6px;
+        height: 6px;
+
+        transform: rotate(-45deg);
+
+        border-top: 2px solid;
```

Comment:
A small thing but we could explicitly set the colour for the borders, at the moment it's being inherited from the toggle element which could change.